study.md

```diff
@@ -57,7 +58,8 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+create
+I'm not 100% sure to be honest... I mean, I feel like it should persist and I haven't seen anything saying otherwise, unless I misunderstood something in the reading... 
```

Comment:
`#create` will instantiate a new object and save it to the database.

`#new` instantiates a new object but doesn't save it to the database.  `#save` needs to be invoked on it afterward to save it.